bin/markdownlint

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eu
+
+markdownlint_version=0.23.0
+
+if ! [ -x "$(command -v markdownlint)" ]; then
```

Comment:
is there a way to verify that the installed verison of markdownlint matches the expected version here?

Reply:
The script checks for the version now as well. If either `markdownlint` is completely missing, or `markdownlint -V` does not equal `markdownlint_version`, it is installed.